cocos/editor-support/spine/SkeletonRenderer.cpp

```diff
@@ -127,14 +127,23 @@ void SkeletonRenderer::initWithJsonFile (const std::string& skeletonDataFile, sp
 
 void SkeletonRenderer::initWithJsonFile (const std::string& skeletonDataFile, const std::string& atlasFile, float scale) {
 	_atlas = spAtlas_createFromFile(atlasFile.c_str(), 0);
-	CCASSERT(_atlas, "Error reading atlas file.");
-
+    
+    	if (!_atlas) {
```

Comment:
Conditional expressions are the opposite of assertions, if you have an “if”, you should not use an assertion and vice versa. You changed the code in such a way that it became generally incorrect.